app/assets/stylesheets/components/gitter_floating.scss

```diff
@@ -1,21 +1,53 @@
-.chat-floating-btn {
-  background: $button-primary;
-  border-radius: 50%;
+.floating-dropdown {
   bottom: 20px;
-  box-shadow: 0 0 15px 2px rgba(0,0,0,0.2);
-  cursor: pointer;
-  padding: 1em;
   position: fixed;
-  right: 20px;
   z-index: 1000;
+  right: 20px;
+
+  .floating-dropdown-item {
+    background: $button-primary;
+    border-radius: 50%;
+    box-shadow: 0 0 15px 2px rgba(0,0,0,0.2);
+    padding: 1em;
+    cursor: pointer;
+
+    &:hover {
+      box-shadow: 0 0 15px 6px rgba(0,0,0,0.2);
+    }
 
-  &:hover {
-    box-shadow: 0 0 15px 6px rgba(0,0,0,0.2);
+    &.main-floating-btn {
+      position: relative;
+      z-index: 2;
+
+      .fa.fa-comments {
+        color: $white;
+        font-size: 2em;
+      }
+    }
+
+    &.chat-floating-btn, &.forum-floating-btn {
```

Comment:
This would be considered a modifier in BEM, we could change this to use one class instead of two.

It seems to be a hidden modifier so I think thats the naming we should use.
something like `.floating-dropdown--hidden-item` for the class in the markup.

Then we can change this line to `&--hidden-item {`